runtime/vm/jvminit.c

```diff
@@ -1899,6 +1899,27 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 			vm->vmRuntimeStateListener.runtimeStateListenerState = J9VM_RUNTIME_STATE_LISTENER_UNINITIALIZED;
 			vm->vmRuntimeStateListener.vmRuntimeState = J9VM_RUNTIME_STATE_ACTIVE;
 
+			argIndex = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXUSECONTAINERSUPPORT, NULL);
+			argIndex2 = FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXDONTUSECONTAINERSUPPORT, NULL);
+
+			if (argIndex > argIndex2) {
+				OMRPORT_ACCESS_FROM_J9PORT(vm->portLibrary);
+				uint64_t subsystemsEnabled = omrsysinfo_cgroup_enable_subsystems(OMR_CGROUP_SUBSYSTEM_ALL);
+
+				if (OMR_CGROUP_SUBSYSTEM_ALL != subsystemsEnabled) {
+					char subsystemsNotEnabled[LARGE_STRING_BUF_SIZE] = "";
```

Comment:
Rather than printing this as strings which need to be updated each time we enable support for more subsystems, can you print the `omrsysinfo_cgroup_get_available_subsystems()` and the `omrsysinfo_cgroup_get_enabled_subsystems()` return values as hex?  

This makes it check the values in the tracefile without having to remember to update the strings.

Reply:
That makes sense. But I don't have to call `omrsysinfo_cgroup_get_enabled_subsystems()` as earlier call to `omrsysinfo_cgroup_enable_subsystems()` returns the same value.